src/Request/RequestHandler.php

```diff
@@ -146,6 +154,8 @@ public function __construct(array $config)
         $this->tokenVerifier = $config['verifier'];
         $this->signatureMethod = $config['sign_with'];
 
+        $this->privateKey = isset($config['private_key']) ? $config['private_key'] : null;
```

Comment:
I think that this should rather declare the privateKey as null by default, and then

```php
if (isset($config['private_key'])) {
    $this->privateKey = $config['private_key'];
}
```